roles/KubevirtTemplateValidator/defaults/main.yml

```diff
@@ -3,3 +3,5 @@
 wait: true
 validator_image: kubevirt-template-validator
 validator_version: "{{ validator_tag | default('latest') }}"
+replicas: "{{ template_validator_replicas | default(2) }}"
```

Comment:
this is ok because it is a "private" detail of the validator deployment, hence should sit here and not in the top-level `_defaults.yml`